app/src/main/java/com/jerboa/ui/components/login/LoginViewModel.kt

```diff
@@ -93,32 +95,45 @@ class LoginViewModel : ViewModel() {
                     ).show()
                 }
                 is ApiState.Success -> {
-                    val luv = siteRes.data.my_user!!.local_user_view
-                    val account = Account(
-                        id = luv.person.id,
-                        name = luv.person.name,
-                        current = true,
-                        instance = instance,
-                        jwt = jwt,
-                        defaultListingType = luv.local_user.default_listing_type.ordinal,
-                        defaultSortType = luv.local_user.default_sort_type.ordinal,
-                    )
+                    val siteVersion = siteRes.data.version
+                    if (compareVersions(siteVersion, MINIMUM_API_VERSION) < 0) {
+                        Toast.makeText(ctx, "Server version too low.", Toast.LENGTH_SHORT).show()
```

Comment:
Needs a translation

Reply:
Not sure how to do that since stringResource needs a Composable block. That's why I didn't use the AlertDialog here.

I can look into it again though, there ought to be a solution

Reply:
I think `ctx.getString(R.string...` is how you do it. 

https://stackoverflow.com/a/7454485

Reply:
Thanks. It's fixed now.